lib/tasks/fix_nil_letting_allocation_values.rake

```diff
@@ -0,0 +1,26 @@
+desc "Infer nil letting allocation values as no"
+task fix_nil_letting_allocation_values: :environment do
+  logs = LettingsLog.where(cbl: nil)
+           .or(LettingsLog.where(chr: nil))
+           .or(LettingsLog.where(cap: nil))
+           .or(LettingsLog.filter_by_year(2024).where(accessible_register: nil))
```

Comment:
Is this an issue for single log submitted ones as well, or just BU? Cause could filter this down by creation_method?